packages/cloud-functions/uma-tvl-calculator/index.js

```diff
@@ -0,0 +1,27 @@
+const { Datastore } = require("@google-cloud/datastore");
```

Comment:
Not relevant for this PR, but having a git-ops style configuration for these might be cool -- where the config part gets committed and how the deployment gets triggered might take some thought, but it could have some nice advantages.

Reply:
for sure! future improvement :)